src/multilspy/language_server.py

```diff
@@ -497,7 +503,8 @@ async def request_completions(
 
             num_retries = 0
             while response is None or (response["isIncomplete"] and num_retries < 30):
-                await self.completions_available.wait()
+                if self.language_id != 'python':
```

Comment:
Would it instead be better to go to the langserver file for these language servers, and just mark the self.completions_available as True in the langserver setup?